src/js/flow/acquia_lift.variations.js

```diff
@@ -53,18 +53,15 @@
   };
```

Comment:
One of the things about agile (at least to me) is what we are planning to do in the near future... may not happen when we expect or at all.  I'd like to make this a simple boolean since it makes no sense to indicate the trigger type.  It is still semantically accurate as a boolean.  You access this by choosing to create an element variation (in the what) screen.  After you enter the URL, this submit function kicks in to set the variable that Drupal can read upon reload to initialize the DOM selector.